vm-virtio/src/transport/pci_device.rs

```diff
@@ -260,6 +272,20 @@ impl VirtioPciDevice {
         self.queue_evts.as_slice()
     }
 
+    /// Sets the event this device uses to interrupt the VM when the used queue is changed.
+    pub fn set_interrupt_evt(&mut self, irq_evt: EventFd) {
```

Comment:
We need to add some support for MSI/MSI-X here, otherwise we will loose this capability from our virtio devices.